### PR TITLE
Remove the "policy firehose"

### PR DESCRIPTION
**Pull request by tijmenb**
This removes the policy firehose (/government/policies/all).

The [policies firehose finder](https://www.gov.uk/government/policies/allhttps://www.gov.uk/government/policies/all) was created by the Elections team during the 2015 General Election, but they didn't have time to test it. So the finder was deployed to production, but not linked to from any other page on GOV.UK, with the eventual aim of replacing the original [policies finder](https://www.gov.uk/government/policies).

As part the of the Government Scrutineers discovery, the policies firehose finder was tested against the original policies finder.

The results of the test are documented here: https://gov-uk.atlassian.net/wiki/spaces/GOVUK/pages/168755249/Analysis+for+Scrutineers

TL;DR The original finder performed slightly better than the newer firehose finder, so the decision has been taken to remove that finder.

https://trello.com/c/YDda1PqI 

cc @leenagupte

### CODE REVIEW

**Comment by leenagupte** [comment]
@tijmenb Before merging and publishing this, the smokey tests need to be updated. The [test](https://github.com/alphagov/smokey/blob/master/features/finder_frontend.feature#L30) that checks for non-blank people could probably be moved the to Whitehall feature.
